server/routes/confirmAchieveGoal/ConfirmAchieveGoalController.ts

```diff
@@ -0,0 +1,87 @@
+import { NextFunction, Request, Response } from 'express'
+import { NewGoal } from '../../@types/NewGoalType'
+import { GoalStatus } from '../../@types/GoalType'
+import locale from './locale.json'
+import validateRequest from '../../middleware/validationMiddleware'
+import transformRequest from '../../middleware/transformMiddleware'
+import ConfirmAchieveGoalPostModel from './models/ConfirmAchieveGoalPostModel'
+import { requireAccessMode } from '../../middleware/authorisationMiddleware'
+import { AccessMode } from '../../@types/Handover'
+import { HttpError } from '../../utils/HttpError'
+import URLs from '../URLs'
+import { goalStatusToTabName } from '../../utils/utils'
+
+export default class ConfirmAchieveGoalController {
```

Comment:
because I am simple, can we have a comment at the top of this and the AchieveGoalController please with what each is for? I'm sure we will forget in 6 months' time :)

Reply:
Yep, not a problem